dumper/app/src/main/resources/oracle-stats/cdb/native/db-links.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  Links.owner "Owner",
```

Comment:
Could you add `CON_ID` column to the db-links file? I forgot to mention it in the ticket because there was no information about this column in the documentation. I checked it now with `desc cdb_db_links;` command and this column is there.

Reply:
Done, I added `ConId` - it's based on the column for cdb and it equals null for dba.